multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -320,30 +330,38 @@ fn make_init_config(chain: Chain) -> DbInitializationConfig {
 fn assert_balances(
     wallet: &Wallet,
     blockchain_interface: &BlockchainInterfaceWeb3<Http>,
-    expected_eth_balance: &str,
-    expected_token_balance: &str,
+    expected_eth_balance: i128,
+    expected_token_balance: i128,
 ) {
-    let eth_balance = blockchain_interface
-        .lower_interface()
-        .get_transaction_fee_balance(&wallet)
-        .unwrap_or_else(|_| panic!("Failed to retrieve gas balance for {}", wallet));
+    let eth_balance = i128::try_from(
+        blockchain_interface
+            .lower_interface()
+            .get_transaction_fee_balance(&wallet)
+            .unwrap_or_else(|_| panic!("Failed to retrieve gas balance for {}", wallet)),
+    )
+    .expect("Uncountable ETH fortune");
     assert_eq!(
-        format!("{}", eth_balance),
-        String::from(expected_eth_balance),
-        "Actual EthBalance {} doesn't much with expected {}",
         eth_balance,
-        expected_eth_balance
+        expected_eth_balance,
+        "Actual EthBalance {} exceeds the expected {} by {}",
+        eth_balance,
+        expected_eth_balance,
+        eth_balance - expected_eth_balance
     );
-    let token_balance = blockchain_interface
-        .lower_interface()
-        .get_service_fee_balance(&wallet)
-        .unwrap_or_else(|_| panic!("Failed to retrieve masq balance for {}", wallet));
+    let token_balance = i128::try_from(
+        blockchain_interface
+            .lower_interface()
+            .get_service_fee_balance(&wallet)
+            .unwrap_or_else(|_| panic!("Failed to retrieve masq balance for {}", wallet)),
+    )
+    .expect("More tokens than they are");
     assert_eq!(
         token_balance,
-        web3::types::U256::from_dec_str(expected_token_balance).unwrap(),
-        "Actual TokenBalance {} doesn't match with expected {}",
+        expected_token_balance,
+        "Actual TokenBalance {} exceeds the expected {} by {}",
         token_balance,
-        expected_token_balance
+        expected_token_balance,
+        token_balance - token_balance
```

Comment:
@bertllll did you mean this instead?

```suggestion
        token_balance - expected_token_balance
```